### ISSUE TITLE
Add ability to pause / resume requestAnimationFrame

### ISSUE THREAD

**Issue by corymsmith** [potree/potree#395]
This would open the door to creating a Potree component in React or other client side frameworks and allow hot module reloading as we would be able to mount / unmount Potree.

**Comment by martinheidegger** [comment]
@corymsmith This seems like a sufficiently easy task for a first pull request ;) _Though it might be but one of many tasks needed for hot-module reloading._

**Comment by corymsmith** [comment]
Yeah I figured I'd wait until you had #391 ready (https://github.com/potree/potree/pull/391) before I added this since its just another thing you'd have to port over.

**Comment by DanielSmon** [comment]
@corymsmith 
> Yeah I figured I'd wait until you had #391 ready (#391) before I added this since its just another thing you'd have to port over.
I was thinking the same thing :)
